Index: app/src/main/java/com/lkb/demo/MainActivity.kt
===================================================================
diff --git a/app/src/main/java/com/lkb/demo/MainActivity.kt b/app/src/main/java/com/lkb/demo/MainActivity.kt
--- a/app/src/main/java/com/lkb/demo/MainActivity.kt	
+++ b/app/src/main/java/com/lkb/demo/MainActivity.kt	
@@ -1,12 +1,7 @@
 package com.lkb.demo
 
 import android.os.Bundle
-import com.google.android.material.snackbar.Snackbar
 import androidx.appcompat.app.AppCompatActivity
-import androidx.navigation.findNavController
-import androidx.navigation.ui.AppBarConfiguration
-import androidx.navigation.ui.navigateUp
-import androidx.navigation.ui.setupActionBarWithNavController
 import com.lkb.demo.databinding.ActivityMainBinding
 
 class MainActivity : AppCompatActivity() {
